ibis/common/exceptions.py

```diff
@@ -45,6 +48,37 @@ class RelationError(ExpressionError):
     """RelationError."""
 
 
+class FieldsNotFoundError(AttributeError, IbisError):
```

Comment:
Why did you decide to subclass `AttributeError` here?

Reply:
This was because it originally was a `FieldNotFoundError` (singular), so it made more sense in the sense of "you tried to lookup some statically known thing on this object that doesn't exist". But now that it stores multiple missing fields, the heirarchy doesn't make sense, so I removed AttributeError as a base class, now it is just inherits from IbisError.